### DISCUSSION TITLE
How to Clean Blocks for Cockpit using fail2ban?

### DISCUSSION THREAD

**Discussion by TS-02** [fail2ban/fail2ban#3249] [Q&A]
Hi.

I am running T-Pot and with it comes Cockpit.

I have been advising to clean the blocks using **fail2ban-client**. 

Please could someone offer some advice on how to do this.

**Comment by sebres** [comment]
What do you mean as "clean the blocks" related to fail2ban?
If you need a filter see https://github.com/cockpit-project/cockpit/issues/722 or provide the log excerpts with failed attempts to be banned.

**Comment by TS-02** [comment]
Hi @sebres,

I asked someone from the T-Pot community and they recommended that I 'cleaned the blocks' for **failban** as **failban** is used for one of their side applications - Cockpit, when logging in.

Could you recommend anything?

**Reply by sebres** [reply]
Again, I don't know what "cleaning the blocks" would mean (in sense of fail2ban). 
And I'm not familiar with Cockpit (admin web UI?) or T-POT (honeypot system?), so I'd somehow reference which blocks are meant and how or why and when they can or must be cleaned.

**Reply by TS-02** [reply]
Okay. Thanks for your help @sebres.
